samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -136,6 +136,50 @@ public async Task<string> ExtractUserIntentAsync(SKContext context)
         return $"User intent: {result}";
     }
 
+    /// <summary>
+    /// Extract the list of participants from the conversation history.
+    /// Note that only those who have spoken will be included.
+    /// </summary>
+    [SKFunction("Extract audience list")]
+    [SKFunctionName("ExtractAudience")]
+    [SKFunctionContextParameter(Name = "chatId", Description = "Chat ID to extract history from")]
+    public async Task<string> ExtractAudienceAsync(SKContext context)
```

Comment:
Why is this necessary? How does it change the behavior if you were to just reference the users list tied to each conversation?